kent2cky/GeoSearch
# Ft style app ui #6

**What does the PR do?**

Style the user interface of the app

**Description of Tasks to be completed**

Have the page looking nice and the interface interactive and 
easy to work with.

**How should this be manually tested?**

After cloning the repo,` cd` into it and run `npm install`
After that, run the command `gulp` and it should be able to open up age page
on your web browser.
Check if the page displays well across various screen sizes

**Any background context you want to provide?**

nil

**What are the relevant pivotal tracker stories?**

#3
